### ISSUE TITLE
pyGIserver failing to start from a fresh repo clone

### ISSUE THREAD

**Issue by chron0** [apollo-ng/PiGI#2]
It seems that pyGIserver won't start from a clean clone anymore, hanging indefinitely at:

```
2014-03-04 21:07:56,298 INFO root: Starting pyGIserver
2014-03-04 21:07:56,301 WARNING pyGI.geigercounter: Could not initialize GPIOs, geigercounter operation will only be simulated!
2014-03-04 21:07:56,350 INFO pyGI.geigerlog: Getting last totalcount
```

@yggi: I could reproduce the problem by deleting my log/geiger_log.db and then restarting the pyGIserver.py.

**Comment by chron0** [comment]
Put a workaround into 92aa21ce6093a5a813f9d9832ff3733ed0267e65 that is not the best of fixes :)

**Comment by chron0** [comment]
fixed with: 29ea72bb5d17320ff4218d2500c71bd211b78e57
